docs/06-concepts/06-database/08-transactions.md

```diff
@@ -18,3 +20,40 @@ var result = await session.db.transaction((transaction) async {
 ```
 
 In the example we insert a company and an employee in the same transaction. If any of the operations fail, the entire transaction will be rolled back and no changes will be made to the database. If the transaction is successful, the return value will be `true`.
+
+## Transaction isolation
+
+The transaction isolation level can be configured when initiating a transaction. The isolation level determines how the transaction interacts with concurrent database operations. If no isolation level is supplied, the level is determined by the database engine.
+
+:::info
+
+At the time of writing, the default isolation level for the PostgreSQL database engine is `IsolationLevel.readCommitted`.
+
+:::
+
+To set the isolation level, configure the `isolationLevel` property of the `TransactionSettings` object:
+
+```dart
+var result = await session.db.transaction(
+  (transaction) async {
+    await Company.db.insertRow(session, company, transaction: transaction);
+    await Employee.db.insertRow(session, employee, transaction: transaction);
+    
+    return true;
+  },
+  settings: TransactionSettings(isolationLevel: IsolationLevel.serializable),
+);
+```
+
+In the example we set the isolation level to `IsolationLevel.serializable`.
+
+The available isolation levels are:
+
+| Isolation Level | Value                 | Description |
```

Comment:
Help requested: `Value` feels like the wrong title but I couldn't think of a better name. Do you have a better suggestion?

Reply:
Maybe `Constant` or `Enum value`?

Reply:
I went with constant: bd9f5ea4f2067b7f343c320c4de0cc1b8489944d